portfolio/src/main/webapp/style.css

```diff
@@ -58,6 +58,21 @@ body {
   text-decoration: none;
 }
 
+.image-grid-row {
+  display: flex;
```

Comment:
I probably missed this earlier, but your flexbox contains exactly one item (at 100% size) so the flexbox doesn't actually do anything here. Indeed, if I remove the flexbox-related properties from your CSS, the webpage acts identical to before.